docs/jwst/barshadow/arguments.rst

```diff
@@ -1,5 +1,12 @@
 Step Arguments
```

Comment:
Note to reviewers: these updates to the barshadow, flatfield, and photom docs are not directly related to the LRS pathloss implementation, but I noticed while updated docs that these steps didn't list their available parameters.